src/core/reducer.test.js

```diff
@@ -387,6 +387,47 @@ test('undo / redo', () => {
   expect(state.G).toEqual({ A: true });
 });
 
+
+test('undo / redo on 2 players game', () => {
```

Comment:
Can you group this test and the undo test right above this under one `describe`?